<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTask</class>
 <widget class="QDialog" name="ViewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о заказе</string>
  </property>
  <widget class="QLineEdit" name="isFulfilledLine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>224</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contentLine">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>224</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>84</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
	border-top: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>39</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Содержание</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнено</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateTaskButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #dbdcf7;
	border: 1px solid #999999;
}

QPushButton:hover {
	background-color: #d5d6f0;
	border: 1px solid #acacac;
}

QPushButton:pressed {
	background-color: #cbcde6;
	border: 2px solid #999999;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
